docs/reference-manual/common-practices/common-practices.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title: Common practices
+nav_order:
+has_children: true
+
+parent: Reference manual
+
+permalink: /docs/reference-manual/common-practices
+---
+
+# Common practices
+
+The Following pages will guide you through reocuring patterns and snipets that can be useful in many different use-cases.
```

Comment:
- reocuring => reoccuring
- snipets => snippets
-

Reply:
done in f0feab7